src/UdApi.ts

```diff
@@ -139,6 +139,10 @@ export default class Udapi extends NamingService {
     return (await this.resolve(domain)).records || {};
   }
 
+  getDomainFromTokenId(tokenId: string): Promise<string> {
+    throw new Error('Method not implemented.');
```

Comment:
Nitpick: as a user, I'd prefer a typed exception in case I ever need to match this error. And I'd also remove the dot in the end.